api/v1beta1/common_types.go

```diff
@@ -128,6 +128,9 @@ const (
 	// MachineSkipRemediationAnnotation is the annotation used to mark the machines that should not be considered for remediation by MachineHealthCheck reconciler.
 	MachineSkipRemediationAnnotation = "cluster.x-k8s.io/skip-remediation"
 
+	// RemediateMachineAnnotation is the annotation used to mark machines that should be remediated by MachineHealthCheck reconciler.
+	RemediateMachineAnnotation = "cluster.x-k8s.io/remediate-machine"
```

Comment:
Naming to be finalized

Reply:
What about `cluster.x-k8s.io/force-remediation` (to express the intent) or `cluster.x-k8s.io/manual-remediation` (similar, not sure if equally clear)

Reply:
I thought a bit about it. I think I like the current annotation the most:

* It's consistent with `cluster.x-k8s.io/delete-machine`
* `cluster.x-k8s.io/force-remediation` for me implies that we force a remediation in a sense that we skip something that would be usually done. Given that the idea here is not skip the usual MHC safeguards I think "force" might be misleading to some users.
* `cluster.x-k8s.io/manual-remediation` I think this unnecessarily limits the semantics of the annotation. We don't actually care if a users sets this annotation manually or e.g. if some other health check system detects an issue and then sets the annotation (without any manual user action)